components/multi-platform-controller/production/host-config.yaml

```diff
@@ -391,14 +391,27 @@ data:
   host.power-rhtap-prod-10.secret: "ibm-production-ppc64le-ssh-key"
   host.power-rhtap-prod-10.concurrency: "1"
 
-  host.sysz-rhtap-prod-1.address: "169.63.184.30"
-  host.sysz-rhtap-prod-1.platform: "linux/s390x"
-  host.sysz-rhtap-prod-1.user: "root"
-  host.sysz-rhtap-prod-1.secret: "ibm-production-s390x-ssh-key"
-  host.sysz-rhtap-prod-1.concurrency: "4"
-
-  host.sysz-rhtap-prod-2.address: "169.59.164.141"
-  host.sysz-rhtap-prod-2.platform: "linux/s390x"
-  host.sysz-rhtap-prod-2.user: "root"
-  host.sysz-rhtap-prod-2.secret: "ibm-production-s390x-ssh-key"
-  host.sysz-rhtap-prod-2.concurrency: "4"
+  # host.sysz-rhtap-prod-1.address: "169.63.184.30"
```

Comment:
Let's remove the lines instead of commenting them, we can always get them back from git history if needed

Reply:
done!